alfresco-transform-misc/alfresco-transform-misc/src/main/resources/misc_engine_config.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Is this a copy - can we remove the duplicate as part of this PR or raise tech debt ?

Reply:
There is a copy in the boot test resources - this is ugly, but it is the fallout of having the configuration in the skinny jar, and the tests that use it in the boot jar, we need more time to refactor, or just move it to the boot jar

Reply:
This is where the file has been renamed to from engine_config.json to misc_engine_config.json
could be an issue with line endings that shown it as a new file.